v2/datastream-to-spanner/pom.xml

```
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
-            <version>20160810</version>
+            <version>20201013</version>
```

Should this also be using the patched version `20231013`?
```suggestion
            <version> 20231013 </version>
```

---

Oops, yeah, I'll update that, but we don't use any of the v2 templates anyhow, so it wouldn't affect any testing I did.  I just made the change here to match (and botched the date)